tests/test_template_types.py

```diff
@@ -2636,3 +2636,19 @@ def test_broadcast_message_single_counts_diacritics_in_gsm(
     })
     assert template.encoded_content_count == 1
     assert template.max_content_count == 1_395
+
+
+@pytest.mark.parametrize('template_class, value_1, value_2', [
+    (EmailPreviewTemplate({'content': "((date&time))", 'subject': 'subject', 'template_type': 'email',
+                          'redact_missing_personalisation': False}), '((date&amp;time))', 'date&time'),
+    (SMSPreviewTemplate({"content": "((>than))", "subject": "((subject))",
+                        "template_type": "sms"}), '((&gt;than))', '>than')
+])
+def test_special_html_characters_used_in_placeholders_are_handled_appropriately(
+        template_class,
+        value_1,
+        value_2
```

Comment:
Can we think of a better name for these? `sanitized_content` and `placeholder_name` maybe?

I think this test would more clearly demonstrate what it's trying to if we also render the replaced text, to show that placeholders go in unescaped but correctly replace the escaped placeholder names.

Reply:
@samuelhwilliams I have removed this test. There is a test in `test_field.py`, `test_replacement_of_placeholders`, that is more appropriate. I have updated it to include tests for html special entities.